bitbadges/chain.json

```diff
@@ -0,0 +1,100 @@
+  "pretty_name": "BitBadges",
+  "chain_id": "bitbadges-1",
+  "bech32_prefix": "cosmos",
+  "daemon_name": "bitbadgeschaind",
+  "node_home": "$HOME/.bitbadgeschaind",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ubadge",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.00025,
+        "average_gas_price": 0.0025,
+        "high_gas_price": 0.025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ustake"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/bitbadges/bitbadgeschain/",
+    "recommended_version": "v1.0-mainnet",
+    "compatible_versions": [
+      "v1.0-mainnet"
+    ],
+    "versions": ["v1.0-mainnet"],
+    "binaries": {
+      "linux/amd64": "https://github.com/BitBadges/bitbadgeschain/releases/download/v1.0-mainnet/bitbadgeschain-linux-amd64",
+      "linux/arm64": "https://github.com/BitBadges/bitbadgeschain/releases/download/v1.0-mainnet/bitbadgeschain-linux-arm64"
+    },
+    "cosmos_sdk_version": "v0.50.8",
+    "ibc_go_version": "v8.3.2",
+    "consensus": {
+      "type": "cometbft",
+      "version": "github.com/cometbft/cometbft v0.38.9"
+    },
+    "genesis": {
+      "genesis_url": "https://github.com/BitBadges/bitbadgeschain/blob/master/genesis.json"
+    },
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitbadges/images/bitbadgeslogo.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitbadges/images/bitbadgeslogo.svg",
+  "description": "BitBadges is the all-in-one, multi-chain platform for building your digital identity.",
+  "peers": {
+    "seeds": [],
+    "persistent_peers": [
+      {
+        "id": "8d1ca31723a66b9058651220fd8da4618bc6f67c",
+        "address": "134.122.12.165:26656"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "http://134.122.12.165:26657",
+        "provider": "bitbadges"
+      },
+    ],
+    "rest": [
+      {
+        "address": "http://134.122.12.165:1317",
+        "provider": "bitbadges"
+      },
+    ],
+    "grpc": []
+  },
+  "explorers": [
+    {
+      "kind": "mintscan",
+      "url": "https://www.mintscan.io/bitbadges",
+      "tx_page": "https://www.mintscan.io/bitbadges/transactions/${txHash}",
+      "account_page": "https://www.mintscan.io/bitbadges/accounts/${accountAddress}"
+    },
+    {
+      "kind": "ping.pub",
+      "url": "https://explorer.bitbadges.io/BitBadges%20Mainnet",
+      "tx_page": "https://explorer.bitbadges.io/BitBadges%20Mainnet/txs/${txHash}",
+      "account_page": "https://explorer.bitbadges.io/BitBadBitBadges%20Mainnetges/account/${accountAddress}"
+    },
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitbadges/images/bitbadgeslogo.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitbadges/images/bitbadgeslogo.svg"
```

Comment:
the png and the svg are not visually idendical, and therefore cannot be placed inside the image object. You may have two image objects in the array.

Reply:
This was not resolved

Reply:
I recommend using the version without text, is it will not be readable >99% of the time.

Reply:
there are still several instances of bitbadges svg and png in the same object. If they are going to remain appearing different, then they should be given different names, and both should be registered as an images for something (either the chain or an asset) at least once.

Reply:
@JeremyParish69 Do I need an SVG, or am I okay to just have the PNG? The PNG is the main official logo. 

In the most recent version, I removed all SVG references.

Reply:
Generally an SVG is better than PNG because it is scalable.
Also, I find it odd that this is perhaps the only token logo with text in it--that is highly unusual because it will not be legible in most UIs. I thought the SVG without text would be more appropriate....
Regardless, I suppose we can proceed as is.

Reply:
Yea, you are right. Thinking about it now, that is a little unconventional, but the logo with the text is kinda how we have been displaying it everywhere.